user/curtail/template.py

```diff
@@ -0,0 +1,13 @@
+pkgname = "curtail"
+pkgver = "1.10.0"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["desktop-file-utils", "gettext", "meson", "pkgconf"]
+makedepends = ["gtk4-devel"]
+depends = ["jpegoptim", "libwebp-progs", "oxipng", "pngquant", "python-scour"]
```

Comment:
this is missing libadwaita and python-gobject

Reply:
i don't mean in makedepends, it's a runtime dependency; this is a python application and opens it via gobject. the -devel's don't do anything here :) 

but fixed it for you, thanks, forgot to add oxipng before...

Reply:
oh! didnt know that was a runtime one!